test/parallel/test-cluster-dgram-1.js

```diff
@@ -111,5 +111,5 @@ function worker() {
     }
   }, PACKETS_PER_WORKER));
 
-  socket.bind(common.PORT);
+  socket.bind(0);
```

Comment:
@benjamingr, from the dgram documentation:

> The options object may contain an additional exclusive property that is use when using dgram.Socket objects with the cluster module. When exclusive is set to false (the default), cluster workers will use the same underlying socket handle allowing connection handling duties to be shared. When exclusive is true, however, the handle is not shared and attempted port sharing results in an error.

When a worker binds a `UDP` socket in non-exclusive mode,[ it requests the master process for a valid socket handle](https://github.com/nodejs/node/blob/master/lib/dgram.js#L208-L230) by [sending a `queryServer` cluster message with the socket info ](https://github.com/nodejs/node/blob/master/lib/internal/cluster/child.js#L50-L88)(key). The master will create the socket handle only after receiving the first message with that specific key and then send the handle back to the worker. The following messages from the workers requesting a socket handle with the same key, will cause the master to send the same handle back to every worker.